bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -540,7 +540,7 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
             case e: ExchangeSubmission               => Partners.name(e.partner)
             case s: SpecialSubmission                => s.specialType.value
             case l: LargeProgramSubmission           => "large program"
-            case i: SubaruIntensiveProgramSubmission => "Subaru intensive program"
```

Comment:
Trivial, but I'm wondering why intensive is capitalized if the other cases aren't.